Lykke.WalletApiv2.Tests/DITests/DITests.cs

```diff
@@ -29,7 +29,7 @@ public DITests()
 
             var containerBuilder = new ContainerBuilder();
 
-            containerBuilder.RegisterModule(new Api2Module(new SettingsServiceReloadingManager<APIv2Settings>("http://settings.lykke-settings.svc.cluster.local/rr5999apiv2999dvgsert25uwheifn_WalletApiv2").Nested(x => x.WalletApiv2)
+            containerBuilder.RegisterModule(new Api2Module(new SettingsServiceReloadingManager<APIv2Settings>("http://settings.lykke-settings.svc.cluster.local/rr5999apiv2999dvgsert25uwheifn_WalletApiv2")

```

Comment:
I suppose this file should be deleted. It's not much use and contains connection string.